docs/bots-guide.md

```diff
@@ -253,8 +252,23 @@ running it manually.
    The standard output of the bot server will be logged to the path in
    your *supervisord* configuration.
 
+### Running a bot in your Zulip Development Environment
+
+For experimental purposes, you may find it easier and safer to develop and run your bot in your local server rather than on a public server like chat.zulip.org.
+So, to run a bot within your local environment, the steps are as follows:
+
+1.  Start up your Development Environment (http://localhost:9991)
+2.  Go to Settings >> Your Bots and create a bot
+3.  Click the icon below your bot name to download the .zuliprc file to your computer
+4.  create a corresponding bot file and place it in zulip_bots/bots directory which you cloned earlier.
+5.  Optionally you could put a ```print('hello bot')```as a way to test your bot.
+ 
+Now,to spin up your bot, assuming you are in the zulip_bots directory, 
+you would do `zulip-run-bot --path-to-bot <bot/botdirectory/botname.py> --config-file ~/.zuliprc.`
```

Comment:
So I think that the steps outlined here are mostly very similar to those in the section `how to run a bot`. How about, rather than creating a new section for the development server setup, we extract the new info covered by this paragraph, and move it to `how to run a bot`? E.g., I could imagine making our refernce to the dev server a link pointing to `http://localhost:9991`. Does that sound good?

Reply:
@derAnfaenger yes it does sounds good. I'll move it into `how to run a bot` instead.